### DISCUSSION TITLE
Error al compilar

### DISCUSSION THREAD

**Discussion by juanse0601** [INGCOM-UNRN-P1/cursada-2023#65] [Preguntas y respuestas] [answered]
Estoy usando visual studio y a la hora de compilar aparece que el archivo no existe
![error](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141684656/08f893fc-04c9-4239-b8a4-cf9def5d976f)

**Comment by martinvilu** [comment]
Abrí una terminal donde estas compilando y ejecutalo de manera directa para ver si el archivo compilado existe.

**Comment by juanse0601** [comment]
![error2](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141684656/da2d0603-d6c4-4f9f-8d1a-ff67cae8b5a4)

**Reply by Clirussa** [reply, marked as answer]
Fijate que es más o menos lo que yo pregunté. Ponelo en el escritorio y al archivo nombralo como C.

**Reply by martinvilu** [reply]
Hay otro problema, revisa que la configuración del path siga estando, en Windows 10 y 11, aparentemente esto no queda configurado entre reinicios
